alica_tests/autogenerated/include/Authority/AuthorityTestMaster1414403396328.h

```diff
@@ -43,6 +45,6 @@ class UtilityFunction1414403396328 : public BasicUtilityFunction
 };
 class PreCondition1414403842622 : public DomainCondition
 {
-    bool evaluate(std::shared_ptr<RunningPlan> rp);
+    bool evaluate(std::shared_ptr<RunningPlan> rp, const IAlicaWorldModel* wm);
```

Comment:
@veerajsk  can you rebase the pr? guess world model changes aren't merged to feature/task_instantiation yet.